tests/conftest.py

```diff
@@ -175,6 +178,49 @@ def release(
     return release
 
 
+@pytest.fixture
+def local_release(
+    release: 'libioc.Release.ReleaseGenerator',
+    root_dataset: libzfs.ZFSDataset,
+    zfs: libzfs.ZFS
+) -> 'libioc.Release.ReleaseGenerator':
+    """Mock a local release."""
+    if not release.fetched:
+        release.fetch(fetch_updates=True, update=True)
+
+    yield release
+    del release
+
+
+@pytest.fixture(scope="function")
+def new_jail(
+    host: 'libioc.Host.Host',
+    logger: 'libioc.Logger.Logger',
+    zfs: libzfs.ZFS,
+) -> 'libioc.Jail.Jail':
+    jail_name = "new-jail-" + str(random.randint(1, 32768))
+    new_jail = libioc.Jail.Jail(
+        dict(name=jail_name),
+        new=True,
+        host=host,
+        logger=logger,
+        zfs=zfs
+    )
+    yield new_jail
+    new_jail.destroy(force=True)
+
+
+@pytest.fixture(scope="function")
+def bridge_interface() -> str:
+    bridge_name = "bridgeTest" + str(random.randint(1024, 4096))
```

Comment:
again, this might be better done with Hypothesis: https://hypothesis.readthedocs.io/en/latest/data.html